File: examples/simp/Python/evaluator/evaluator6.py
# Import the library
import LinkPredPython as lpp
# Create an evuator object
ev = lpp.Evaluator();
ev.addADA();
# Load scores from pst.txt
ev.addPST("PST", "pst.txt");
ev.addRAL();
ev.addPR();
ev.addTPR();
ev.run("Zakarays_Train.edges", "Zakarays_Test.edges");

File: examples/simp/CPP/evaluator/evaluator6.cpp
#include <linkpred.hpp>
#include <iostream>

using namespace LinkPred::Simp;

int main() {
	Evaluator eval;
	eval.addADA();
	eval.addPST("PST", "pst.txt"); // Load scores from pst.txt
	eval.addRAL();
	eval.addPR();
	eval.addTPR();
	eval.run("Zakarays_Train.edges", "Zakarays_Test.edges");
	return 0;
}
